api/models/helpers.py

```diff
@@ -0,0 +1,29 @@
+import abc
+
+from django.db.backends.sqlite3.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+class DataMigrationMaster(abc.ABC):
+    """
+    Abstract class-based variation used to make data-migrations.
+    See Django docs: https://docs.djangoproject.com/en/stable/topics/migrations/#data-migrations
+    """
+
+    def __init__(self, apps: StateApps, schema_editor: DatabaseSchemaEditor):
+        self.apps = apps
+        self.schema_editor = schema_editor
+
+    @abc.abstractmethod
+    def main(self) -> None:
```

Comment:
```suggestion
    def _populate(self) -> None:
```
Both `main()` and `run()` in the same class are a bit confusing. I'd suggest making this method "private" to make it absolutely clear which one should be passed to `RunPython`.